docs/getting-started/dev-portal/html.md

```diff
@@ -1,15 +1,8 @@
-# Setup
+# Elements Dev Portal in HTML
 
+Setting up Elements Dev Portal is a case of loading up some JavaScript and CSS and embedding a Web Component (custom HTML) into the HTML of your new or existing dev portal. This should work anywhere custom HTML can be embedded and this approach is supported by [pretty much all modern browsers](https://caniuse.com/custom-elementsv1).
 
-
-## Web Component
-
-Setting up Elements Dev Portal is a case of loading up some JavaScript and CSS and embedding a Web Component (custom HTML) into the HTML of your existing documentation. 
-
-This works with pretty much any CMS: Drupal, Joomla, Wordpress, or Jekyll. They all allow custom HTML to be embedded and this approach is supported by [pretty much all modern browsers](https://caniuse.com/custom-elementsv1).
-
-For now, the only component in the DevPortal package is `<elements-stoplight-project/>`, which lets users render Stoplight Projects wherever they want. There will eventually be more types of dev portal in the package. 
-
+For now, the only component in the DevPortal package is `<elements-stoplight-project/>`, which lets users display Stoplight Projects wherever they want. There will eventually be more types of dev portal in the package. 
```

Comment:
```suggestion
For now, the only component in the Dev Portal package is `<elements-stoplight-project/>`, which lets users display Stoplight Projects wherever they want. There will eventually be more types of dev portal in the package. 
```

Also not sure when 'dev portal' should and should not be in title case, but I noticed it's not consistent if that matters.